pkg/webhook/policy.go

```diff
@@ -171,9 +181,15 @@ func (h *validationHandler) Handle(ctx context.Context, req atypes.Request) atyp
 		if vResp.Response.Result == nil {
 			vResp.Response.Result = &metav1.Status{}
 		}
+
+		if h.reporter != nil {
+			h.reporter.ReportRequest(req.AdmissionRequest, vResp.Response, time.Since(timeStart))
```

Comment:
Could all of these be folded into something like:

```
defer func() {
		if h.reporter != nil {
			h.reporter.ReportRequest(req.AdmissionRequest, vResp.Response, time.Since(timeStart))
}()
```

?

Reply:
thanks, moved to defer with response suggestion from @ekitson